pkg/plugins/plugins.go

```diff
@@ -269,6 +269,15 @@ func (scanner *PluginScanner) loadPluginJson(pluginJsonFilePath string) error {
 		return errors.New("did not find type or id properties in plugin.json")
 	}
 
+	if pluginCommon.Id == "gel" {
+		isEnabled := scanner.cfg.FeatureToggles["expressions"]
```

Comment:
setting.IsExpressionsEnabled()

Reply:
This is where you want to add what Kyle said: https://github.com/grafana/grafana/blob/b027e25127c1aa38a4bd845ca63708824dab8b55/pkg/plugins/transform_plugin.go#L30

Reply:
@kylebrandt @marefr Why do we prefer the [global](https://github.com/grafana/grafana/blob/master/pkg/setting/setting.go#L1172) `FeatureToggles` variable over the one in [Cfg](https://github.com/grafana/grafana/blob/master/pkg/setting/setting.go#L287)? I mean the comment above Cfg says "// TODO move all global vars to this struct".

Reply:
I'm not sure about global vs cfg, but I do think I method is nicer than a string in a map.

Reply:
@kylebrandt Yeah, but it's more important to use the right value :)

Reply:
You can move `IsExpressionsEnabled` to `Cfg` if you want.